src/workloads/query/LimitSkip.yml

```diff
@@ -0,0 +1,134 @@
+- classic
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 5
+  LimitSkipNumber: &limitSkipNum {^Inc: {start: 1, step: 100}}
+
+ActorTemplates:
+- TemplateName: RunAggCommand
+  Config:
+    Name: {^Parameter: {Name: "name", Default: "RunAggCommand"}}
+    Type: CrudActor
+    Database: *Database
+    Collection: *Collection
+    Threads: 4
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "active", Default: [0]}}
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1000
+          Database: *Database
+          Collection: *Collection
+          Operations:
+          - OperationMetricsName: sbe
+            OperationName: aggregate
+            OperationCommand:
+              Pipeline: {^Parameter: {Name: "pipeline", Default: []}}
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 400
+
+Actors:
+# Clear any pre-existing collection state.
+- Name: ClearCollection
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: *Threads
+        Collection: *Collection
+        Operations:
+        - OperationName: drop
+
+- Name: InsertData
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          zero: 0
+          hello: "hellohellohellohellobye"
+
+# Phase 2: Ensure all data is synced to disk.
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+
+# Phase 3-N: Run pipelines with limit, skip, and limit + skip.
+- ActorFromTemplate:
+    TemplateName: RunAggCommand
+    TemplateParameters:
+      name: "LimitVaryConstant"
+      active: [3]
+      pipeline: [{$match: {$and: [{hello: "hellohellohellohellobye"}, {zero: 0}]}}, {$limit: *limitSkipNum}, {$project: {_id: 0}}]
+
+- ActorFromTemplate:
+    TemplateName: RunAggCommand
+    TemplateParameters:
+      name: "SkipVaryConstant"
+      active: [4]
+      pipeline: [{$match: {$and: [{hello: "hellohellohellohellobye"}, {zero: 0}]}}, {$skip: *limitSkipNum}, {$project: {_id: 0}}]
+
+- ActorFromTemplate:
+    TemplateName: RunAggCommand
+    TemplateParameters:
+      name: "LimitSkipVaryConstant"
+      active: [5]
+      pipeline: [{$match: {$and: [{hello: "hellohellohellohellobye"}, {zero: 0}]}}, {$skip: *limitSkipNum}, {$limit: *limitSkipNum} , {$project: {_id: 0}}]
+
+AutoRun:
+- When:
```

Comment:
Let's also add a pipeline with sort, as it should absorb limit and also should be parameterized.

Reply:
Done.

Reply:
Done. I added `sort -> limit -> skip` and `sort -> skip -> limit`.